### ISSUE TITLE
Athena practice brands missing aliases

### ISSUE THREAD

**Issue by AnalogJ** [fastenhealth/fasten-sources#33]
It looks like Middleton Family Medicine has a related practice called "[New England House Calls](https://9808.portal.athenahealth.com/)"

Only "New England House Calls" shows up in the list, Middleton is missing because its not shown as an alias in the brand. 

Also, from the Main portal (https://9808.portal.athenahealth.com/ vs https://9808-2.portal.athenahealth.com/ ), the brand name should be "MIDDLETON FAMILY MEDICINE"

**Comment by athena-hendrix** [comment]
Just to give a little background on this, you may have discovered this while a practice was going through a brand migration. The combination of the two urls you linked in this post should not lead to two different brands (and if you go to https://9808.portal.athenahealth.com/ you now get a selector).

I will say though that if Fasten isn't properly supporting athena brands then that is a potential issue.

**Comment by AnalogJ** [comment]
Yeah I had to do some pretty awful screenscraping because the old Athena customer list (csv) only had the endpoint information. 

Looks like you have a full FHIR bundle with metadata now (I'm guessing its HTI1/HTI2 compliant?) 

https://service-base-urls.api.fhir.athena.io/athena-fhir-service-base-urls.json

I'll take a look at that.

**Comment by AnalogJ** [comment]
looks like its still missing logos, aliases and branding info (websites). 
Any chance that'll be added soon @athena-hendrix ?
